### PR TITLE
ci(core): add server, web and apk build execution

### LINKED ISSUE #63: Add build test execution in CI
Add build execution in CI for web, mobile and back.

(add mobile apk build only on main)

### PR DESCRIPTION
**Pull request by Miou-zora**
Linked to:
- close #63

### CODE REVIEW

File: .github/workflows/apk_check.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# name: Apk check
+
+# # on pr on main branch
+# on:
+#   pull_request:
+#     branches:
+#       - main
+
+# jobs:
```
Comment (Benjicatch) [inline .github/workflows/apk_check.yml:9]: we gonna reput this one day ?

Comment (Miou-zora) [reply .github/workflows/apk_check.yml:9]: If one day, we decide to add apk check in ci, we can reuse this file soooo

**Review by Benjicatch — APPROVED** [review]
ok

**Review by Pablodeibar — APPROVED** [review]
looks good

**Pull request merged**